assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -140,7 +136,9 @@ class Question extends React.Component {
                   }}
                   value={this.state.postBody}
                   maxLength={300}
-                  ref={(t) => { this.txtarea = t; }}
+                  ref={(t) => {
+                    this.txtarea = t;
```

Comment:
you should take a look at the doc about refs : https://facebook.github.io/react/docs/refs-and-the-dom.html
You don't need these very often but sometimes it's very useful. It's a reference to a DOM element or an instance of a class component. So you can trigger an event on an element when another event occur on another element